### DISCUSSION TITLE
When are the latest releases updated in Conda-Forge

### DISCUSSION THREAD

**Discussion by Benjas86** [quarto-dev/quarto-cli#8649] [Q&A] [answered]
### Description

Hello, this is hopefully a simple question. I'm fairly stuck on how I can use Quarto due to working restrictions, currently using Anaconda and have the current Quarto version available from conda-forge. When is version 1.4 likely to be released on conda-forge please. Thank you

**Comment by mcanouil** [comment, marked as answer]
As far as I know Quarto team does not maintain this:
- https://github.com/quarto-dev/quarto-cli/discussions/5328

See https://github.com/conda-forge/quarto-feedstock

Note the opened issue:
- https://github.com/conda-forge/quarto-feedstock/issues/35

CC @mfisher87

**Reply by Benjas86** [reply]
Thank you, I'll keep an eye over on that discussion. 1.4 is a great update.

**Reply by mfisher87** [reply]
Thanks Quarto team for sending folks over to the feedstock repo and for looping me in! I've been _super_ excited to get 1.4 out (I don't know why, I'm weirdly excited about the "breadcrumbs" feature), but life hasn't cooperated in giving me the time yet. I will try again this weekend!

If anyone wants to get started sooner, I'd be happy to support them. We can always use more Conda Forge maintainers, and IMO it's easy to get started. Just post in https://github.com/conda-forge/quarto-feedstock/issues/35 :)
